kaal-presentation/build.gradle.kts

```diff
@@ -45,7 +46,11 @@ android {
     }
 
     lintOptions {
-        setLintConfig(rootProject.file("lint.xml"))
+        lintConfig = rootProject.file("lint.xml")
+    }
+
+    buildFeatures {
+        dataBinding = true
```

Comment:
The dependency to the databinding should be only a dependency on `androidx.databinding:databinding-runtime` because the databinding feature is not used in this module.

Reply:
Changed to compileOnly dependency on Release version 4.0.1 of databinding. Is that ok?